### PR TITLE
Add show_expired to Variable#value

### PR DESCRIPTION
**Pull request by apotterri**

### CODE REVIEW

File: lib/conjur/variable.rb [hunk 1]
```diff
@@ -200,9 +200,13 @@ def version_count
     #
     # @param [Integer] version the **1 based** version.
     # @return [String] the value of the variable
-    def value(version = nil)
+    def value(version = nil, show_expired = false)
```
Comment (kgilpin) [inline lib/conjur/variable.rb]: I wonder if `show_expired` should be an options hash.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# v4.21.1 (unreleased)
```
Comment (kgilpin) [inline CHANGELOG.md]: This would be `4.22.0`

File: lib/conjur/variable.rb (hunk 1 above)
Comment (apotterri) [inline lib/conjur/variable.rb]: Ok, makes sense to make it a hash. I also realized I forgot to document it, I'll add that, too.

File: CHANGELOG.md (hunk 2 above)
Comment (apotterri) [inline CHANGELOG.md]: OK

**Pull request merged**